mne_connectivity/spectral/epochs.py

```diff
@@ -760,6 +760,7 @@ def _check_estimators(method, mode):
     return con_method_types, n_methods, accumulate_psd, n_comp_args
 
 
+@verbose
 @fill_doc
```

Comment:
for all of these, `@verbose` can simply replace `@fill_doc` (`@verbose` does everything `@fill_doc` does plus a few other things)

Reply:
There are some places where both `@verbose` and `@fill_doc` are present - should I remove the latter there too?

Reply:
actually now that I look, I realize that MNE-Connectivity has it's own `@fill_doc` definition, but it uses MNE-Python's `@verbose`.  Off the top of my head I don't know how that will interact, so there's a chance you might actually need both.  I recommend only changing these 3 here, then checking the results before changing other occurrences elsewhere in the codebase.

Reply:
(also you will need to import verbose in those files where you're adding it)

Reply:
If you put fill_doc below verbose, I think it should work
```
@verbose
@fill_doc
def whatever(....)
````

Reply:
yes, this is what is done elsewhere